src/components/organisms/AdminVenueView/components/DateTimeField/DateTimeField.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback, useEffect, useState } from "react";
+
+import { getDateTimeFromUtc, getUtcFromDateTime } from "utils/time";
+
+import "./DateTimeField.scss";
+
+export interface DateFieldProps {
+  title: string;
+  subTitle?: string;
+  name: string;
+  dateTimeValue?: number;
+  handleDateTimeChange: (args: number) => void;
```

Comment:
Since this is the public interface of the component, something more in line with common React style

```suggestion
  onChange: (value: number) => void;
```